### ISSUE TITLE
Create CRUD for environment variables 

### ISSUE THREAD

**Issue by girl-loves-coding** [softflow24/data-river#121]
### Duplicates

- [X] I have searched the existing issues

### Summary 💡

We can take inspiration from Postmen and how they handle env variables, but do not copy it 1:1. (see screenshot). As we already have a side bar to configure nodes, lets reuse it to add environment variables
1. An icon to open a side bar to add a new environment variables
2. A form to add a variable
3. An icon to save the variable
4. Add an option to select available environment variables
<img width="991" alt="Na5rblfz4S" src="https://github.com/user-attachments/assets/a63b1ae6-705e-479f-a5ae-7bfb423c1cd7">

### Examples 🌈

Look up how they do it for Postmen

**Comment by girl-loves-coding** [comment]
How to: 
Look into these files, https://github.com/softflow24/data-river/blob/main/packages/editor/src/components/controls/top-right-controls/RunWorkflowButton.tsx#L45-L48 and https://github.com/softflow24/data-river/blob/main/packages/execution-engine/src/VariableResolver.ts . There is already some implemented functionality

**Comment by girl-loves-coding** [comment]
We have  already implemented side bar, please re-use it and make the UI for env variables similar to the one on the screenshot

**Comment by girl-loves-coding** [comment]
Inspiration:

![image](https://github.com/user-attachments/assets/58105fd1-eab7-4f38-8d35-eee70b2c2d07)
![image](https://github.com/user-attachments/assets/ea80b70d-6777-43bd-8aee-162138a7eb3c)
![image](https://github.com/user-attachments/assets/a3abe95d-3dac-4f69-9a6e-52782804a893)

**Comment by girl-loves-coding** [comment]
yes, all looks good. A couple of details to the last screenshot: 
1. Remove Save icon. Save methos will be triggered from Save variable button at the bottom.
2. rename: Filter param -> fiilter var
<img width="881" alt="Screenshot 2024-11-12 150601" src="https://github.com/user-attachments/assets/281f333b-f9e0-4d5c-ae32-488db9b7f893">
iables; Add New Param -> Add new variable; Save configuration -> save variable

**Comment by IraBoh** [comment]
14.11.24 
Plan:
- [ ] Check the implented requiements
- [ ] Write down if changes required
- [ ] Define next steps
- [ ] Write DOTOs for the steps
